src/logic/chatGPT.js

```diff
@@ -0,0 +1,45 @@
+const axios = require("axios");
+import { key } from '../../key';
+
+
+export let chatResponse = '';
+
+export const createOpenAIAxiosClient = () => {
+  const apiKey = key;
+
+  const client = axios.create({
+    headers: {
+      Authorization: "Bearer " + apiKey,
+    },
+  });
+
+  return client;
+};
+
+export const sendMessage = (userQuestion) => {
+  const openAIAxios = createOpenAIAxiosClient();
+
+  const params = {
+    messages: [
+      { role: "system", content: "you are the all-seeing eye.ś" },
+      { role: "user", content: userQuestion },
+    ],
+    model: 'gpt-3.5-turbo',
+    max_tokens: 100,
+    temperature: 0,
+  };
+
+  
+  return openAIAxios.post("https://api.openai.com/v1/chat/completions", params)
+    .then((results) => {
```

Comment:
try to use async await instead. Promise is not but just i want you to practice async/await a little bit.